### PR TITLE
Unbreaks dev and beta CI

### LINKED ISSUE #792: Fix master branch CI
The most recent [CI run](https://github.com/flutter/samples/runs/2495320467) is failing:

```
~/work/samples/samples/animations ~/work/samples/samples
Running "flutter pub get" in animations...                       1,128ms
Analyzing animations...                                         

  error • A value of type 'Animation<BorderRadius?>' can't be assigned to a variable of type 'Animation<BorderRadius>' • lib/src/misc/repeating_animation.dart:27:21 • invalid_assignment
```

### PR DESCRIPTION
**Pull request by RedBrogdon**
Updates the CI scripts in `/tool` to match the updates recently used to fix CI in the beta branch.

Fixes #792.

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

**Comment by RedBrogdon** [comment]
@domesticmouse, you're on vacation and my tests haven't even passed yet. 😄

**Comment by RedBrogdon** [comment]
CC: @johnpryan, I just want to make sure you see this before it's merged, since it's meant to unblock some of the stuff you're working on. Feel free to land it whenever you want.

**Review by johnpryan — APPROVED** [review]

**Pull request merged**
